test/expected/size_utils.out

```diff
@@ -150,7 +150,7 @@ $$);
 SELECT * FROM chunks_detailed_size('toast_test');
      chunk_schema      |    chunk_name    | table_bytes | index_bytes | toast_bytes | total_bytes | node_name 
 -----------------------+------------------+-------------+-------------+-------------+-------------+-----------
- _timescaledb_internal | _hyper_4_9_chunk |       24576 |       16384 |        8192 |       49152 | 
+ _timescaledb_internal | _hyper_4_9_chunk |        8192 |       16384 |       24576 |       49152 | 
```

Comment:
Why did these sizes exchange positions? It looks like a typo in the view.

Reply:
I verified this against the table. Earlier toast size was calculated based on the _hypertable's_ reltoastrelid and not the chunk's reltoastrelid. That's the reason for different toast size.  This in turn changes the heap size.

Reply:
Ok. Think you need to make it very clear in the commit comment that this is a bug fix and that the previous size was wrong. Otherwise, I'm fine.